boot/bootutil/include/bootutil/crypto/ecdsa.h

```diff
@@ -622,43 +623,45 @@ static inline int bootutil_ecdsa_parse_public_key(bootutil_ecdsa_context *ctx,
 #endif /* MCUBOOT_USE_MBED_TLS */
 
 #if defined(MCUBOOT_USE_NRF_EXTERNAL_CRYPTO)
-typedef uintptr_t bootutil_ecdsa_p256_context;
-
-static inline void bootutil_ecdsa_p256_init(bootutil_ecdsa_p256_context *ctx)
+typedef uintptr_t bootutil_ecdsa_context;
+static inline void bootutil_ecdsa_init(bootutil_ecdsa_context *ctx)
 {
     (void)ctx;
 }
 
-static inline void bootutil_ecdsa_p256_drop(bootutil_ecdsa_p256_context *ctx)
+static inline void bootutil_ecdsa_drop(bootutil_ecdsa_context *ctx)
 {
     (void)ctx;
 }
 
-static inline int bootutil_ecdsa_p256_verify(bootutil_ecdsa_p256_context *ctx,
-                                             uint8_t *pk, size_t pk_len,
-                                             uint8_t *hash,
-                                             uint8_t *sig, size_t sig_len)
+static inline int bootutil_ecdsa_verify(bootutil_ecdsa_context *ctx,
+                                        uint8_t *pk, size_t pk_len,
+                                        uint8_t *hash, size_t hash_len,
+                                        uint8_t *sig, size_t sig_len)
 {
     (void)ctx;
     (void)pk_len;
+    (void)hash_len;
     uint8_t dsig[2 * NUM_ECC_BYTES];
 
     if (bootutil_decode_sig(dsig, sig, sig + sig_len)) {
         return -1;
     }
 
-	/* As described on the compact representation in IETF protocols,
-	 * the first byte of the key defines if the ECC points are
-	 * compressed (0x2 or 0x3) or uncompressed (0x4).
-	 * We only support uncompressed keys.
-	 */
-	if (pk[0] != 0x04)
-		return -1;
+    /* Only support uncompressed keys. */
+    if (pk[0] != 0x04) {
+        return -1;
+    }
```

Comment:
the changes here can be squashed to original commit/fixup!

Reply:
OK, can do that.

Reply:
Fixed